lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -105,6 +126,14 @@ def test_empty_topic_id(self):
         self.form_data.setlist("topic_id", ["", "not empty"])
         self.assert_error("topic_id", "This field cannot be empty.")
 
+    @ddt.data(*itertools.combinations(["topic_id", "text_search"], 2))
```

Comment:
Yeah, this is a bit of future-proofing because we are about to add more things to this list. This will just make for a smaller diff for the next piece.